src/views/ScorePage/ScorePage.js

```diff
@@ -59,17 +81,17 @@ function getCurrentScore(Answers) {
 
 function nicknameValidation() {
   const input = document.getElementById('nickname');
-  if (input.value != '' && input.value != 'NICKNAME') {
+  if (input.value !== '' && input.value !== 'NICKNAME') {
     return true;
-  } else {
-    return false;
   }
+  return false;
 }
 
 function saveQuizScore() {
   const quizScores = JSON.parse(localStorage.getItem('quizScores')) || [];
   const recentUserScore = getCurrentScore(userAnswers);
   if (!nicknameValidation()) {
+    // eslint-disable-next-line no-alert
```

Comment:
Let's remove this part
leave just condition when nicknameValidation() is true

additionally, please implement a function that disables the "submit" button when there is no user's nickname in the input field